pkg/config/config.go

```diff
@@ -245,6 +260,24 @@ func parseEnv(cfg *Config) error {
 		cfg.InitialAdminKeys = append(cfg.InitialAdminKeys, initialAdminKeys...)
 	}
 
+	// split allowed headers and append to cfg
+	if allowedHeadersEnv := os.Getenv("SOFT_SERVE_HTTP_CORS_ALLOWED_HEADERS"); allowedHeadersEnv != "" {
+		allowedHeaders := strings.Split(allowedHeadersEnv, " ")
+		cfg.HTTP.CORS.AllowedHeaders = append(cfg.HTTP.CORS.AllowedHeaders, allowedHeaders...)
+	}
+
+	// split allowed origins and append to cfg
+	if allowedOriginsEnv := os.Getenv("SOFT_SERVE_HTTP_CORS_ALLOWED_ORIGINS"); allowedOriginsEnv != "" {
+		allowedOrigins := strings.Split(allowedOriginsEnv, " ")
+		cfg.HTTP.CORS.AllowedOrigins = append(cfg.HTTP.CORS.AllowedOrigins, allowedOrigins...)
+	}
+
+	// split allowed methods and append to cfg
+	if allowedMethodsEnv := os.Getenv("SOFT_SERVE_HTTP_CORS_ALLOWED_METHODS"); allowedMethodsEnv != "" {
+		allowedMethods := strings.Split(allowedMethodsEnv, " ")
+		cfg.HTTP.CORS.AllowedMethods = append(cfg.HTTP.CORS.AllowedMethods, allowedMethods...)
+	}
+
```

Comment:
`caarlos0/env` should split these variables by comma `,` so there's no need to do that here.